talisker/endpoints.py

```diff
@@ -247,7 +260,222 @@ def metrics(self, request):
         return Response(data, status=200, mimetype=CONTENT_TYPE_LATEST)
 
     @private
-    def debug_logtree(self, request):
+    def logtree(self, request):
+        """Display the stdlib logging configuration."""
         import logging_tree
         tree = logging_tree.format.build_description()
         return Response(tree)
+
+    @private
+    def packages(self, request):
+        """List of python packages installed."""
+        import pkg_resources
+        pkgs = list(pkg_resources.working_set)
+        pkgs.sort(key=lambda p: p.project_name)
```

Comment:
why do we explicitly convert to a list?

ISTM this just needs to be iterable?

i.e. `for p in sorted(pkg_resources.working_set, key=lambda p: p.project_name):`